src/components/patientsList/item/Item.js

```diff
@@ -12,34 +12,30 @@ import {
 } from "../../../constants/styles/typography";
 import PatientBasicData from "../../../constants/propTypes/basicDataPropTypes";
 
-const Item = ({ onPress, item: patient, data: patientBasicData }) => {
+const Item = ({ onPress, patient, patientsBasicData }) => {
   return (
     <View style={styles.item}>
       <TouchableOpacity onPress={onPress}>
         <Header patient={patient} />
         <InterviewInfo
           icon="diagnosis"
           name="Diagnoza"
-          value={
-            patient.diagnosis ? patient.diagnosis.join(", ") : patient.code
-          }
+          value={patient.diagnosis ? patient.diagnosis.join(", ") : ""}
```

Comment:
Dlaczego znikło patient.code?

Reply:
To musimy jeszcze przedystkutować. Patient w bazie danych nie ma pola code, dlatego że w moim zamyśle, wszystkie diagnozy przechowujemy w tabeli Diagnosis. Chyba, że chcemy jakoś szczególnie wyróżnić tę diagnozę, to wtedy możemy coś takiego umieścić.